lms/djangoapps/commerce/utils.py

```diff
@@ -44,6 +44,13 @@ class EcommerceService(object):
     def __init__(self):
         self.config = CommerceConfiguration.current()
 
+    def toggle_commerce_config(self, enabled=False, checkout_page='/test_basket'):
+        """ Enable / Disable CommerceConfiguration model """
+        CommerceConfiguration.objects.create(
+            checkout_on_ecommerce_service=enabled,
+            single_course_checkout_page=checkout_page
+        )
```

Comment:
Should this be the following?
```
self.config.checkout_on_ecommerce_service = enabled
self.config.single_course_checkout_page = checkout_page
self.config.save()
```
Also, I would rename this to `update_commerce_config`.